src/scripts/contract_allocations.py

```diff
@@ -0,0 +1,55 @@
+from src.constants import NODE_URL
+from src.fetch.contracts import EvmAccountInfo
+from src.files import AllocationFiles, NetworkFile, File
+from src.generate.merkle_data import MerkleLeaf
+
+
+def non_wallet_allocation(network: str, allocation_file: File) -> int:
+    allocations = MerkleLeaf.load_from(allocation_file)
+    account_info = EvmAccountInfo(
+        node_url=NODE_URL[network],
+        addresses=[a.Account for a in allocations],
+        network=network
+    )
+    contracts = account_info.contracts(load_from=NetworkFile("contracts.txt"))
+    not_wallets = account_info.get_non_wallets(
+        [c for c, is_contract in contracts.items() if is_contract]
```

Comment:
Depending on how many contracts there are it might be worth it to put `contracts.items()` in a set to speed up the lookup here. But in the grand scheme of things it probably doesn't matter.

Reply:
I actually want to refactor how contracts are stored. Nowhere yet in this project have we ever used the false case so yes, a set would be a more appropriate return type with equivalent information. However I would prefer to do this in a separate PR.

Reply:
I'll create an issue for this and start on it ASAP.

Reply:
https://github.com/gnosis/cow-token-allocation/issues/15